utils/design-vars.stanza

```diff
@@ -30,5 +29,16 @@ public var MAX-HOLE-SIZE-TOLERANCE:  Double = 0.0508 ; The tolerance on the diam
 public var MIN-HOLE-SIZE-TOLERANCE:  Double = 0.0508 ; The tolerance on the diameter of a hole of the smallest size
 public var HOLE-POSITION-TOLERANCE:  Double = 0.0508 ; The tolerance on placing any hole
 
-; For consolidation
-;public var PREFERRED-MANUFACTURERS = ["Yageo", "Panasonic Electronic Components", "Vishay Dale", "TE Connectivity Passive Product"]
+;============================================
+;====== BOM Generator configuration =========
+;============================================
+
+; Set it to false to allow any distributor
+public var APPROVED-DISTRIBUTOR-LIST = ["Allied Electronics & Automation"
```

Comment:
Duncan mentioned yesterday that this list should be ordered by preference (see [JITX-2040](https://linear.app/jitx/issue/JITX-2040/bom-sort-by-preferred-vendors)).

1. Alphabetical looks pleasing to the eye! But, is there a preferred ordering to these we should be using in the default?
2. The name change from "preferred" to "approved" suggests we're now using as a filter (rather than allowing fallbacks to distributors outside of the list). Is that intended? (if so, we can match that in [JITX-2040](https://linear.app/jitx/issue/JITX-2040/bom-sort-by-preferred-vendors))